mkosi/__init__.py

```diff
@@ -1442,6 +1442,41 @@ def want_signed_pcrs(config: Config) -> bool:
     )
 
 
+def get_ukify_secure_boot_arguments(context: Context) -> tuple[list[PathString], list[PathString]]:
```

Comment:
This feels messy

Can we instead do a `ukify()` function that takes a `Context` parameter as well as extra arguments and options and all it does is add architecture and secure boot stuff before invoking ukify?